.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,36 @@
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+| Name| about |title|labels|
+| ------------- | ------------- |------------- |------------- |
+| Feature Request  | Suggest an idea for this project  |[Short Description] (Version: [AYS]) |enhancement|
+
+**Read before creating an issue**
+In general, we close any issues that are
+* unactionable(fill the template below under "Description")
+* questions best served elsewhere
+We have a small community of people that manages issues, and we want to ensure that the issues that remain open are high quality
```

Comment:
Can you revise this sentence as shown below so that someone can understand the concept better.

`We have a small community of individuals responsible for managing issues, and our goal is to ensure that the open issues maintain a high level of quality.`

Reply:
@Rapter1990 buradaki konu çözülmüşse, resolve edip mergeleyebiliriz.